app/models/work.rb

```diff
@@ -57,17 +60,16 @@ def event_context
     end
 
     event :submit_for_review do
-      transition %i[first_draft version_draft] => :pending_approval
+      transition %i[first_draft version_draft rejected] => :pending_approval
     end
 
     event :reject do
-      transition pending_approval: :first_draft, if: ->(work) { work.druid.blank? }
-      transition pending_approval: :version_draft, if: ->(work) { work.druid.present? }
+      transition pending_approval: :rejected
     end
 
     event :update_metadata do
       transition deposited: :version_draft
-      transition %i[first_draft version_draft] => same
+      transition %i[first_draft version_draft rejected] => same
     end
```

Comment:
note that I specifically asked Amy to ensure that the rejection reason displayed at the appropriate times and that everything else seemed to work ok ...